backend/pkg/api/data_access/vdb_management.go

```diff
@@ -343,56 +341,111 @@ func (d *DataAccessService) GetValidatorDashboardOverview(ctx context.Context, d
 			case constypes.DbExited:
-				data.Validators.Exited += state.Count
+				data.Validators.Exited++
 			}
 		}
+
+		// Find rocketpool validators
+		type RpOperatorInfo struct {
+			Pubkey             []byte           `db:"pubkey"`
+			NodeFee            float64          `db:"node_fee"`
+			NodeDepositBalance *decimal.Decimal `db:"node_deposit_balance"`
+			UserDepositBalance *decimal.Decimal `db:"user_deposit_balance"`
+		}
+		var queryResult []RpOperatorInfo
+		query := `
+			SELECT 
+				pubkey,
+				node_fee,
+				node_deposit_balance,
+				user_deposit_balance
+			FROM rocketpool_minipools
+			WHERE pubkey = ANY($1)`
+
+		err = d.alloyReader.SelectContext(ctx, &queryResult, query, pubKeyList)
+		if err != nil {
+			return fmt.Errorf("error retrieving rocketpool validators data: %w", err)
+		}
+
+		rpValidators := make(map[string]RpOperatorInfo)
+		for _, res := range queryResult {
+			if res.NodeDepositBalance != nil && res.UserDepositBalance != nil {
+				rpValidators[hexutil.Encode(res.Pubkey)] = res
+			}
+		}
+
+		// Create a new sub-dashboard to get the total cl deposits for non-rocketpool validators
+		var nonRpDashboardId t.VDBId
+
+		for _, validator := range validators {
+			metadata := validatorMapping.ValidatorMetadata[validator]
+			validatorBalance := utils.GWeiToWei(big.NewInt(int64(metadata.Balance)))
+			effectiveBalance := utils.GWeiToWei(big.NewInt(int64(metadata.EffectiveBalance)))
+
+			if rpValidator, ok := rpValidators[hexutil.Encode(metadata.PublicKey)]; ok {
+				// Calculate the balance of the operator
+				fullDeposit := rpValidator.UserDepositBalance.Add(*rpValidator.NodeDepositBalance)
+				operatorShare := rpValidator.NodeDepositBalance.Div(fullDeposit)
+				invOperatorShare := decimal.NewFromInt(1).Sub(operatorShare)
+
+				base := decimal.Min(decimal.Max(decimal.Zero, validatorBalance.Sub(*rpValidator.UserDepositBalance)), *rpValidator.NodeDepositBalance)
+				commission := decimal.Max(decimal.Zero, validatorBalance.Sub(fullDeposit).Mul(invOperatorShare).Mul(decimal.NewFromFloat(rpValidator.NodeFee)))
+				reward := decimal.Max(decimal.Zero, validatorBalance.Sub(fullDeposit).Mul(operatorShare).Add(commission))
+
+				operatorBalance := base.Add(reward)
+
+				if protocolModes.RocketPool {
+					data.Balances.Total = data.Balances.Total.Add(operatorBalance)
+				} else {
+					data.Balances.Total = data.Balances.Total.Add(validatorBalance)
+				}
+				data.Balances.StakedEth = data.Balances.StakedEth.Add(*rpValidator.NodeDepositBalance)
+			} else {
+				data.Balances.Total = data.Balances.Total.Add(validatorBalance)
+
+				nonRpDashboardId.Validators = append(nonRpDashboardId.Validators, validator)
+			}
+			data.Balances.Effective = data.Balances.Effective.Add(effectiveBalance)
+		}
+
+		// Get the total cl deposits for non-rocketpool validators
+		if len(nonRpDashboardId.Validators) > 0 {
+			totalNonRpDeposits, err := d.GetValidatorDashboardTotalClDeposits(ctx, nonRpDashboardId)
```

Comment:
I assumed `StakedEth` means CL deposits for non-rocketpool validators.
If that is not the case please comment here.